deepchecks/vision/checks/data_integrity/abstract_property_outliers.py

```diff
@@ -50,6 +52,8 @@ class AbstractPropertyOutliers(SingleDatasetCheck):
         - 'categorical' - for discrete, non-ordinal outputs. These can still be numbers,
```

Comment:
Semi related question, why are properties not just a dict of name: callable?
More specifically would love help understanding the usage of output_type
@yromanyshyn (git say you originally added this)

Reply:
Actually IMO it's @nirhutnik

Reply:
output_type is for the display - 
The display need to know if to plot a KDE graph (numerical) or barchart (categorical). The "class_id" type exists so that we can map the label values (usually integers) to the actual names of the class (using VisionData._label_map)

Reply:
Didn't understand :/ lets talk tomorrow before I start working on NLP properties

Reply:
We display the distribution of each property in checks like PropertyDrift.
For numerical properties, we show a continuous graph:
![image](https://user-images.githubusercontent.com/92314933/182610755-c21a5a0c-ee49-4b82-82b8-bab3a27ce611.png)

For categorical properties, we show a bar graph:
![image](https://user-images.githubusercontent.com/92314933/182610867-c35c98db-b6c6-4772-9c82-ae05429a1ebe.png)

Also, notice that it shows the correct class name in the x-axis.

The only way for the check to know which type of graph to show (and which type of drift to calculate) is by telling it which type of variable it is (numerical/categorical).

Better explanation?